src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Square extends Figure {
```

Comment:
The class 'Square' should not begin with an empty line. Please remove the redundant empty line at the start of the class definition.